### PR TITLE
[CPM-72] Seed programme degree dummy data

### PR DESCRIPTION
**Pull request by wryao64**

### CODE REVIEW

**Review by Richardng12 — APPROVED** [review]
lgtm

File: seed-db/init.sh [hunk 1]
```diff
@@ -2,5 +2,4 @@
 
 mongoimport --collection courses --drop --file fakeCourses.json --jsonArray --uri "mongodb://mongo:27017"
 mongoimport --collection regulations --drop --file fakeRegulations.json --jsonArray --uri "mongodb://mongo:27017"
-
-
+mongoimport --collection programmes --drop --file fakeProgrammes.json --jsonArray --uri "mongodb://mongo:27017"
```
Comment (daltonlim) [inline seed-db/init.sh:5]: Change table name to programmedegrees for API to work

**Review by daltonlim — CHANGES_REQUESTED** [review]

File: seed-db/init.sh [hunk 2]
```diff
@@ -2,5 +2,4 @@
 
 mongoimport --collection courses --drop --file fakeCourses.json --jsonArray --uri "mongodb://mongo:27017"
 mongoimport --collection regulations --drop --file fakeRegulations.json --jsonArray --uri "mongodb://mongo:27017"
-
-
+mongoimport --collection programmeDegrees --drop --file fakeProgrammeDegrees.json --jsonArray --uri "mongodb://mongo:27017"
```
Comment (daltonlim) [inline seed-db/init.sh:5]: do we wanna stick with programmeDegrees or programmedegrees? The api currently requires programmedegrees

Comment (wryao64) [reply seed-db/init.sh:5]: I think programmeDegrees cause it's 2 words

Comment (anian100) [reply seed-db/init.sh:5]: I originally went with programmedegrees cuz URIs dont have caps, but I dont have a strong preference, up to you guys

**Review by daltonlim — APPROVED** [review]
LGTM

**Pull request merged**